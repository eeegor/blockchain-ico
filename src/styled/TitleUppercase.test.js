import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { TitleUppercase } from '.';

describe('TitleUppercase', () => {
	test('renders default h1', () => {
		const tree = renderer
			.create(<TitleUppercase>Title</TitleUppercase>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('text-transform', 'uppercase');
	});

	test('renders h1 with custom color', () => {
		const tree = renderer
			.create(<TitleUppercase color="green">Title</TitleUppercase>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('color', 'green');
	});
});
