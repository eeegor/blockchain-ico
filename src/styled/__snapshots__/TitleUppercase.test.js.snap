// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleUppercase renders default h1 1`] = `
.c0 {
  font-size: 16px;
  line-height: 20px;
  font-family: GTCinetype-Bold;
  margin-bottom: 12px;
  color: #101530;
  text-transform: uppercase;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

<h2
  className="c0"
>
  Title
</h2>
`;

exports[`TitleUppercase renders h1 with custom color 1`] = `
.c0 {
  font-size: 16px;
  line-height: 20px;
  font-family: GTCinetype-Bold;
  margin-bottom: 12px;
  color: #101530;
  text-transform: uppercase;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  color: green;
}

<h2
  className="c0"
  color="green"
>
  Title
</h2>
`;
